_data/tutorials.yaml

```diff
@@ -302,3 +302,14 @@ console-consumer-read-specific-offsets-partition:
     ksql: disabled
     kstreams: disabled
     kafka: enabled
+
+kafka-connect-datagen-local:
+  title: "How to generate mock data to a local Kafka topic using the Kafka Connect Datagen"
+  meta-description: "How to generate mock data to a local Kafka topic using the Kafka Connect Datagen"
+  slug: "/kafka-connect-datagen-local"
+  problem: "you want to test your Kafka applications but need mock data produced to Kafka topics."
+  introduction: "You will run a local instance of the kafka-connect-datagen connector to produce mock data to a local Kafka cluster"
```

Comment:
Would a link to the datagen repository work here?

Reply:
Yes, added